package.json

```diff
@@ -110,7 +110,7 @@
     "date-fns-tz": "^2.0.0",
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
-    "expensify-common": "2.0.57",
+    "expensify-common": "2.0.60",
```

Comment:
This needs to be updated as new version is available now!

Reply:
@allroundexperts I have no issues bumping it in this PR but this version bumped here is specifically related to the changes in the PR. If I bump it here and there are regressions, this PR will be reverted which I don't think would be correct.

Let me know if you agree, otherwise I will bump it 🙂

Reply:
It's sort of a difficult decision. If you don't bump it, and we merge it, the latest version on main might get over-ridden.